SU2_PY/SU2/opt/project.py

```diff
@@ -160,10 +160,11 @@ def __init__( self, config, state=None ,
             # look for existing designs
             folders = glob.glob(self._design_folder)
             if len(folders)>0:
-                sys.stdout.write('Removing old designs in 10s.')
-                sys.stdout.flush()
-                if warn: time.sleep(10)
-                sys.stdout.write(' Done!\n\n')
+                for countdown in range(10, 0, -1):
```

Comment:
"your mission should you choose to accept it..." :)

Reply:
"This simulation will self-destruct in ten seconds. Good luck"